MSAL/src/instance/MSALB2CAuthority.m

```diff
@@ -33,13 +33,12 @@
 @implementation MSALB2CAuthority
 
 - (instancetype)initWithURL:(NSURL *)url
-                    context:(id<MSIDRequestContext>)context
                       error:(NSError **)error
 {
-    self = [super initWithURL:url context:context error:error];
+    self = [super initWithURL:url error:error];
     if (self)
     {
-        self.msidAuthority = [[MSIDB2CAuthority alloc] initWithURL:url context:context error:error];
+        self.msidAuthority = [[MSIDB2CAuthority alloc] initWithURL:url context:nil error:error];
```

Comment:
I think we'll need to remove "tfp" validation for B2C authority

Reply:
done